src/medications/MedicationOptionFields.tsx

```diff
@@ -0,0 +1,47 @@
+import { Option } from '../shared/components/input/SelectWithLableFormGroup'
+import useTranslator from '../shared/hooks/useTranslator'
+
+const MedicationFieldsOptions = () => {
```

Comment:
Currently not used as I am not sure how best to implement this. I wanted a central place where options used in NewMedication and ViewMedication can be stored but when i had this not as a component then i could not get the translation to work since it needs to happen inside a component (i hope that makes sense). Doing translation outside of a component seems to have potential issues? https://github.com/i18next/react-i18next/issues/909

Reply:
I have removed this for now, we can consider refactoring in future.